lms/djangoapps/teams/tests/test_views.py

```diff
@@ -274,7 +311,7 @@ def test_filter_text_search(self):
     @ddt.data(
         (None, 200, ['Nuclear Team', u'sólar team', 'Wind Team']),
         ('name', 200, ['Nuclear Team', u'sólar team', 'Wind Team']),
-        ('open_slots', 200, ['Wind Team', 'Nuclear Team', u'sólar team']),
+        ('open_slots', 200, ['Nuclear Team', u'sólar team', 'Wind Team']),
```

Comment:
There is an additional student used for testing membership endpoints that I added to the 'Wind Team'. This pushed the team further back in the list because it now has fewer open slots.